types/src/v24/wallet/mod.rs

```diff
@@ -250,7 +250,7 @@ pub struct ListUnspentItem {
     /// The bitcoin address of the transaction.
     pub address: String,
     /// The associated label, or "" for the default label.
-    pub label: String,
+    pub label: Option<String>,
```

Comment:
The need for this change proves what we suspect, our test coverage is not good enough (see #58). 

cc @jamillambert